crates/solidity/outputs/cargo/tests/src/bindings_output/runner.rs

```diff
@@ -10,7 +10,8 @@ use slang_solidity::bindings;
 use slang_solidity::cst::KindTypes;
 use slang_solidity::parser::{ParseOutput, Parser};
 
-use super::graph::{render_dot_graph, render_graph};
+use super::graph::graphviz::render as render_graphviz_graph;
+use super::graph::mermaid::render as render_mermaid_graph;
```

Comment:
Unrelated to this PR:

now that we have built-ins, what do you think of using `success` and `failure` file suffixes to indicate the status of binding (instead of parsing), as that is what we are really checking here? i.e. If a file has syntax errors, but we still bind everything else, it is a success.

Thoughts?